src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/Validators/ValidationBehavior.shared.cs

```diff
@@ -169,19 +171,19 @@ public ICommand ForceValidateCommand
 
```

Comment:
Ah - good catch! 

Yea, I wish we had a EditorConfig that would throw an exception when a `Task` is not awaited. Discarding, and not awaiting, a `Task` like this is a huge code smell:
```csharp
// Don't ever do this
_ = Task.Delay(100);
```

Reply:
@brminnick doesn't exist any analyzer that does that? If there's some we can add it on the project.

Reply:
@pictos I'll upgrade [CS4014](https://msdn.microsoft.com/en-us/library/hh873131.aspx) to be an Error, but this only works if the method is `async`.

It looks like there are a few [3rd Party NuGet Packages](https://cezarypiatek.github.io/post/async-analyzers-summary/) we could leverage, but I'd rather implement those in another Issue/PR so that we can give them the a deeper focus before adding them to `develop`/`main`.

Reply:
What the reason to await a task if I don't need it?

Reply:
Because a `Task` will catch every `Exception` that is thrown inside of the itself; i.e. the compiler wraps the code of an `async` method inside of a try/catch block. The `Exception` thrown/caught inside of the `Task` is re-thrown as long as the developer has `await`'d the `Task`. However, non-`await`d `Tasks` do _not_  rethrow `Exception`s. 

We should always `await` every `Task` and every `ValueTask` to ensure the `Exception` is rethrown.

I do into more detail here in the README of my `AsyncAwaitBestPractices` repo if you want to check out what the IL code looks like for `async`/`await`/`Task`: https://github.com/brminnick/AsyncAwaitBestPractices#trycatch-in-movenext